config/routes.rb

```diff
@@ -1,7 +1,12 @@
 Rails.application.routes.draw do
-    root "users#index"
+  root "users#index"
 
-    resources :users, only: %i[index show] do 
-    resources :posts, only: %i[index show]
+  resources :users, only: %i[index show] do 
+    resources :posts, only: %i[index new create show]
+  end
+
+  resources :posts do
+    resources :comments, only: %i[create]
+    resources :likes, only: %i[create]
```

Comment:
- [ ] please provide a corresponding routes to get  comments

Reply:
Hi @DanSam5K, 
You can get comments from posts.